### PR TITLE
Enum

### PR DESCRIPTION
**Pull request by melinamboedecker**

### CODE REVIEW

**Comment by melinamboedecker** [comment]
This PR is ready for reivew. console.logs for these changes have been removed, front end queries/mutations relevent to the enum have been adjusted and are working, seeds adjusted for enum,  so be sure to re-seed (as category and label are now only part of the post and not separate models, I just put categories and labels in the postSeed for some posts, and where there is none specified it will default to None.    I have not yet deleted the models for Category and Label, but with the enum these are no longer needed so let me know if you'd like me to delete them in this PR.     Front and back ends are working on this branch.

File: server/schemas/resolvers/category.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-const Category = require('../../models/Category');
+// const Category = require('../../models/Category');
```
Comment (vasylynash) [inline server/schemas/resolvers/category.js]: We don't need this file? Then just remove it

File: server/schemas/resolvers/label.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
-const Label = require('../../models/Label');
+// const Label = require('../../models/Label');
```
Comment (vasylynash) [inline server/schemas/resolvers/label.js]: Same here

File: server/schemas/resolvers/category.js (hunk 1 above)
Comment (melinamboedecker) [reply server/schemas/resolvers/category.js]: done, thanks

File: server/schemas/resolvers/label.js (hunk 2 above)
Comment (melinamboedecker) [reply server/schemas/resolvers/label.js]: done, thanks

**Review by vasylynash — APPROVED** [review]

**Review by Zoneam — APPROVED** [review]

**Comment by melinamboedecker** [comment]
[Enum](https://trello.com/c/St0641fZ/15-enum)

**Pull request merged**